test/services/reply-comments.test.js

```diff
@@ -0,0 +1,102 @@
+const assert = require('assert');
+const app = require('../../server/app');
+const service = app.service('comments');
+const contributionService = app.service('contributions');
+const userService = app.service('users');
+
+const { userData } = require('../assets/users');
+const { contributionData } = require('../assets/contributions');
+const { commentData } = require('../assets/comments');
+
+const replyData = {
+  content: 'Nested comment'
+};
+
+const replyUserData = {
+  email: 'test3@test3.de',
+  password: '1234',
+  name: 'Jack',
+  timezone: 'Europe/Berlin',
+  badgeIds: [],
+  role: 'user'
+};
+
+
+describe('\'reply comments\' service', () => {
+  let user;
+  let replyUser;
+  let params, replyParams;
+  let contribution;
+  let comment;
+
+  before(function(done) {
+    this.server = app.listen(3031);
+    this.server.once('listening', () => done());
+  });
+
+  after(function(done) {
+    this.server.close(done);
+  });
+
+  beforeEach(async () => {
+    await app.get('mongooseClient').connection.dropDatabase();
+    user = await userService.create(userData);
+    replyUser = await userService.create(replyUserData);
+
+    params = {
+      user
+    };
+    replyParams = {
+      replyUser
+    };
+
+    contribution = await contributionService.create(contributionData, params);
+
+    commentData.userId = user._id;
```

Comment:
Setting the `userId` on a comment or a reply should not be necessary. The user authenticates with the server, so the server should be responsible for the `userId` field.

Reply:
I could change this in the `comments.test.js`, but without it the `reply-comments.test.js` will fail when running it.